### PR TITLE
dont override window.PARSELY if it exists

### PR DESCRIPTION
**Pull request by Wattenberger**

### CODE REVIEW

**Review by msukmanowsky — CHANGES_REQUESTED** [review]
To be on the safe side, shouldn't we also add a defensive check in `gatsby-browser.js` to see if the `PARSELY.pageviewQueue` object is available?

File: src/utils.js [hunk 1]
```diff
@@ -15,9 +15,14 @@ export function getOptions(pluginOptions) {
...
-    location.pathname
-    }${location.search}${location.hash}`;
+  return [
+    window.location.protocol,
+    "//",
+    window.location.host,
+    location.pathname,
+    location.search,
+    location.hash,
+  ].join("")
```
Comment (msukmanowsky) [inline src/utils.js:25]: This change is completely fine ofc but plz try not to slip these kinds of things into PRs about separate issues.

**Comment by Wattenberger** [comment]
added that check 👍

**Comment by msukmanowsky** [comment]
Ok, looks good. Can you resolve conflicts with master?

**Comment by Wattenberger** [comment]
👍 got it, I totally forgot that I had forked the repo

**Pull request merged**